tests/res.test.ts

```diff
@@ -1,10 +1,11 @@
 import test from "ava"
 import { getTestFixture } from "tests/fixtures/get-test-fixture"
 import { su } from "@tscircuit/soup-util"
+import { res } from "src/fn/res"
 
 test("res_imperial0402", async (t) => {
   const { fp, snapshotSoup } = await getTestFixture(t)
-  const soup = fp.string("res_imperial0402").soup()
+  const soup = fp.string("2512").soup()
 
```

Comment:
Why was this changed? Imperial0402

Reply:
oops I missed when I was testing all passive fn